source/SUBMIT.rst

```diff
@@ -126,40 +126,38 @@ Example preprint templates using ``repo2data`` for caching data on NeuroLibre se
    * - Nilearn
      - `neurolibre/repo2data-nilearn <https://github.com/neurolibre/repo2data-caching>`_
    * - OSF
-     - `neurolibre/repo2data-osf <https://github.com/ltetrel/neurolibre-osf-test>`_
+     - `neurolibre/repo2data-osf <https://github.com/neurolibre/neurolibre-osf-test>`_
 
 
 
-.. warning:: RoboNeuro may fail downloading relatively large datasets (**exceeding 5GB**) as the book build process times out in 30 minutes.
+.. warning:: 
+  RoboNeuro may fail downloading relatively large datasets (**exceeding 5GB**) as the book build process times out in 60 minutes.
+  If you have data that exceed 5GB, please create an issue in your github repository so a Neurolibre admin can check it.
 
 .. topic:: Help RoboNeuro find your data during book build
 
-  RoboNeuro downloads your data to a folder named ``data``, which is created at the base of your repository.
+  `Repo2Data <https://github.com/SIMEXP/Repo2Data>`_ downloads your data to a folder named ``data/PROJECT_NAME``, which is created at the base of your repository.
```

Comment:
What do you mean by format ? Basically `PROJECT_NAME` is a variable that depends on the user requirements.

Reply:
>  PROJECT_NAME is a variable that depends on the user requirements.

I meant a naming convention for the PROJECT_NAME like `surname_projectname_year` etc. to ensure that it is unique.

Reply:
Or we could do also is to leave this field optional (in data_req.json), if it is not given it would follow the convention you suggest `username/repo_name`.
Anyway I suggest we keep it as is ATM, and we can think of it later since this is not a huge deal WDYT?

Reply:
Sure, we can proceed as is, at this point functionality is the main concern as you said.